src/ripple/rpc/handlers/AMMInfo.cpp

```diff
@@ -66,7 +66,7 @@ to_iso8601(NetClock::time_point tp)
     return date::format(
         "%Y-%Om-%dT%H:%M:%OS%z",
```

Comment:
The `%Om` should just be `%m`.  Adding the `O` gives the locale's alternative representation.  But this is just the numbers 1-12, and I'd be surprised if we really intend on alternative representation of decimal digits.  Ditto for `%OS`.  It should just be `%S`.

Reply:
I replaced this with `%FT%TZ` and changed type conversion instead, to preserve full-second period of `NetClock`. See last commit  https://github.com/XRPLF/rippled/pull/4775/commits/9f1d544dcbae542d9d1d11ac43d36c5e168eba9a

Reply:
That looks perfect! :-)